src/commands/init.ts

```diff
@@ -29,6 +34,20 @@ export const initCommandDecorator = (command: commander.Command): void => {
         }
         loadConfiguration(opts.file);
 
+        await validatePrereqs(["terraform", "git", "az", "helm"], true);
+
+        await validateAzure(true);
+
+        await validateEnvVariables(
+          [
+            "ARM_SUBSCRIPTION_ID",
```

Comment:
Just a note based on what we discussed yesterday, if we're using static env vars for infra validation, should we not be expecting the same env vars in spk.config?

Reply:
That is still an open discussion. I think for now, we can assume we expect these environment variables to be defined, but if we do not have a reason for it to be stored in the `spk-config.yaml`, we should leave it out. This can obviously change!